### ISSUE TITLE
SteamVR, VR, Virtual Reality games and categories not listed?

### ISSUE THREAD

**Issue by Nerderizer** [rallion/depressurizer#86]
LOVE this project, it's pretty amazing. 

However, one of the main reasons I want this is to properly separate my VR games where Steam and the developers consistently failed to tag their VR games with the store tags for VR. Could you add these VR related categories, tags, and game titles to the system?

I am currently following the instructions to scrap the store, so hopefully it will catch them at that point, if it's possible currently.

**Comment by mvegter** [comment]
@UserID10T This repository is not actve anymore, please checkout my fork:
> [mvegter/Depressurizer](https://github.com/mvegter/Depressurizer)

**Comment by ppriest** [comment]
@mvegter is your repository a fork of https://github.com/Depressurizer/Depressurizer/ ? Since that has many years more development and multiple contributors than Rallion's original and is considered the de facto fork.

**Comment by mvegter** [comment]
@ppriest Yes, mine is a fork of [Depressurizer/Depressurizer](https://github.com/Depressurizer/Depressurizer/), only that one has gone inactive as well. I'm one of the contributers of [Depressurizer/Depressurizer](https://github.com/Depressurizer/Depressurizer/).
